nix/svclib.nix

```diff
@@ -74,6 +46,55 @@ let
       topology = mkShelleyNode;
     in toFile "topology.yaml" (toJSON topology);
 
+  ## mkFullyConnectedLocalClusterLegacyTopologyWithProxy
+  ##   :: Int Port
+  ##   -> [String NodeName]
+  ##   -> String NodeName
+  ##   -> String NodeName -> Int PortNo
+  ##   -> Topology FilePath
+  ##
+  ##  Produce a store path containing a legacy SL topology
+  ##  with following properties:
+  ##   - fully connected between the nodes
```

Comment:
We should include a comment next to the place where the number of legacy nodes is set that one cannot pick too high a number of legacy nodes since we use an "all-to-all" topology and of course that does not scale well. It's ok for 3 or 4, but the old nodes don't handle lots of connections that well.

Reply:
Done.